nx/lib/nx/defn/expr.ex

```diff
@@ -675,6 +675,12 @@ defmodule Nx.Defn.Expr do
     expr(out, tensor.data.context, :cholesky, [tensor])
   end
 
+  @impl true
```

Comment:
What should I have written here? How can I test this piece of code?

Reply:
I'm not sure testing this would be useful. It's up to a backend to handle (or not) an expression correctly. Effectively, what expr/4 is doing is adding the `:qr` step and the given tensors and options to the expression graph.

Reply:
This is fine! You did it right :)